views/js/runner/plugins/content/accessibility/keyNavigation.js

```diff
@@ -356,10 +356,6 @@ define([
             replace : true,
             loop : true,
             elements : navigators,
-        }).on('tab', function(){
-            this.next();
```

Comment:
Aren't the removed code necessary ?
Have you tested if there is no regression of the original story: https://oat-sa.atlassian.net/browse/TAO-3738 ?

Reply:
Added it back